src/chi-vue/src/utils/utils.ts

```diff
@@ -32,3 +34,36 @@ export function contains(ancestorElement: HTMLElement, descendantElement: HTMLEl
     return contains(ancestorElement, descendantElement.parentElement);
   }
 }
+
+export function findComponent(startComponent: Vue, componentName: string) {
+  let component = null;
+  let parent = startComponent.$parent;
+  while (parent && !component) {
+    if (parent.$options.name === componentName) {
```

Comment:
What if parent is a HTML element and not a Vue component? You should use the HTML form of parent instead of the Vue form.